src/linux/cpu.rs

```diff
@@ -790,7 +790,17 @@ pub(crate) fn get_vendor_id_and_brand() -> (String, String) {
     }
     if let (Some(implementer), Some(part)) = (implementer, part) {
         vendor_id = get_arm_implementer(implementer).map(String::from);
-        brand = get_arm_part(implementer, part).map(String::from);
+        // It's possible to "model name" even with an ARM CPU, so just in case we can't retrieve
+        // the brand from "CPU part", we will then use the value from "model name".
+        //
+        // Example from raspberry pi 3B+:
+        //
+        // ```
+        // model name      : ARMv7 Processor rev 4 (v7l)
```

Comment:
Note that it is still wrong, the Raspberry Pi 3B+ has a Cortex-A53, which is an ARMv8 processor, not ARMv7.

Reply:
Absolutely but then it's up to the manufacturer to fix the wrong information they provide, not us. Also, this information is only provided in the case we don't have this model, not by default.